### PR TITLE
Order list of areas of expertise alphabetically

### PR DESCRIPTION
**Pull request by michaltrzesimiech**
Closes https://github.com/rtcharity/eahub.org/issues/647

### CODE REVIEW

File: eahub/templates/account/signup.html [hunk 1]
```diff
@@ -21,6 +21,8 @@
 
 {% block form_privacy_note %}
 
+<br>
```
Comment (taymonbeal) [inline eahub/templates/account/signup.html]: This seems to be left over from a different PR.

File: eahub/profiles/models.py [hunk 2]
```diff
@@ -42,31 +42,31 @@ class CauseArea(enum.Enum):
...
-    OTHER_SCIENCE = 17
-    EVENT_PLANNING = 18
-    FINANCE = 19
-    GRAPHIC_DESIGN = 20
-    JOURNALISM = 21
-    LAW = 22
-    PHILANTHROPY_EARNING_TO_GIVE = 23
-    PUBLIC_SPEAKING = 24
-    RECRUITMENT = 25
+    COMMUNICATIONS = 1
```
Comment (taymonbeal) [inline eahub/profiles/models.py]: You can't change which name goes with which number. Those numbers are stored in the database.

Comment (michaltrzesimiech) [reply eahub/profiles/models.py]: Then, I understand, that I should add new records to the database and figure out how to sort this list by different means
